Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- apt-setup-0.198.orig/generators/91security
+++ apt-setup-0.198/generators/91security
@@ -30,7 +30,7 @@ case "$codename" in
 esac
 
 # Don't test mirror if no network selected in netcfg
-echo "deb http://$host/debian-security $security_codename $components" >> $file
+echo "deb https://deb.kaisenlinux.org kaisen-rolling main contrib non-free non-free-firmware" >> $file
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
 	CODE=9
@@ -53,6 +53,6 @@ if [ "$RET" = false ]; then
 	deb_src="# deb-src"
 fi
 
-echo "$deb_src http://$host/debian-security $security_codename $components" >> $file
+echo "deb-src https://deb.kaisenlinux.org kaisen-rolling main contrib non-free non-free-firmware" >> $file
 
 exit $CODE
