Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- apt-setup-0.198.orig/generators/92updates
+++ apt-setup-0.198/generators/92updates
@@ -36,13 +36,9 @@ echo "# ${codename}-updates, to get upda
 echo "# see https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_updates_and_backports" >> $file
 
 if  [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "deb $protocol://${host}${directory} ${codename}-updates $components" >> $file
+	echo ""
 else
-	echo "# A network mirror was not selected during install.  The following entries" >> $file
-	echo "# are provided as examples, but you should amend them as appropriate" >> $file
-	echo "# for your mirror of choice." >> $file
-	echo "#" >> $file
-	echo "# deb http://deb.debian.org/debian/ ${codename}-updates $components" >> $file
+	echo ""
 fi
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
@@ -71,9 +67,9 @@ if [ "$RET" = false ]; then
 fi
 
 if [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "$deb_src $protocol://${host}${directory} ${codename}-updates $components" >> $file
+	echo ""
 else
-	echo "# deb-src http://deb.debian.org/debian/ ${codename}-updates $components" >> $file
+	echo ""
 fi
 
 exit $CODE
--- apt-setup-0.198.orig/generators/93backports
+++ apt-setup-0.198/generators/93backports
@@ -35,13 +35,9 @@ echo "# ${codename}-backports, previousl
 
 # Don't test mirror if no network selected in netcfg
 if  [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "deb $protocol://${host}${directory} ${codename}-backports $components" >> $file
+	echo ""
 else
-	echo "# A network mirror was not selected during install.  The following entries" >> $file
-	echo "# are provided as examples, but you should amend them as appropriate" >> $file
-	echo "# for your mirror of choice." >> $file
-	echo "#" >> $file
-	echo "# deb http://deb.debian.org/debian/ ${codename}-backports $components" >> $file
+	echo ""
 fi
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
@@ -70,9 +66,9 @@ if [ "$RET" = false ]; then
 fi
 
 if [ -n "$protocol" ] && [ -n "$host" ]; then
-	echo "$deb_src $protocol://${host}${directory} ${codename}-backports $components" >> $file
+	echo ""
 else
-	echo "# deb-src http://deb.debian.org/debian/ ${codename}-backports $components" >> $file
+	echo ""
 fi
 
 exit $CODE
