Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- apt-setup-0.198.orig/generators/50mirror
+++ apt-setup-0.198/generators/50mirror
@@ -52,7 +52,7 @@ if [ -e /cdrom/.disk/base_installable ];
 	if [ -e /cdrom/.disk/cd_type ]; then
 		cd_type=$(cat /cdrom/.disk/cd_type)
 	fi
-	cd_count=$(grep "^deb cdrom:" $ROOT/etc/apt/sources.list.new | wc -l)
+	#cd_count=$(grep "^deb cdrom:" $ROOT/etc/apt/sources.list.new | wc -l)
 
 	no_network=
 	if db_get netcfg/dhcp_options && \
