Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- apt-setup-0.198.orig/finish-install.d/62apt-cdrom-setup
+++ apt-setup-0.198/finish-install.d/62apt-cdrom-setup
@@ -35,7 +35,7 @@ if [ "$disable_cdrom_entries" = "true" ]
 	sed -i "/^deb cdrom:/s/^/#/" /target/etc/apt/sources.list
 	case "$reason" in
 	    small_media)
-		cat >> /target/etc/apt/sources.list <<EOF
+		grep -v -E '^(#|$)' /target/etc/apt/sources.list
 
 # This system was installed using small removable media
 # (e.g. netinst, live or single CD). The matching "deb cdrom"
@@ -45,7 +45,7 @@ if [ "$disable_cdrom_entries" = "true" ]
 EOF
 		;;
 	    not_a_cd)
-		cat >> /target/etc/apt/sources.list <<EOF
+		grep -v -E '^(#|$)' /target/etc/apt/sources.list
 
 # This system was installed using removable media other than
 # CD/DVD/BD (e.g. USB stick, SD card, ISO image file).
@@ -56,7 +56,7 @@ EOF
 EOF
 		;;
 	    preseed)
-		cat >> /target/etc/apt/sources.list <<EOF
+		grep -v -E '^(#|$)' /target/etc/apt/sources.list
 
 # This system was installed using removable media.
 # The matching "deb cdrom" entries were disabled at the end
